src/ontology/cl-edit.owl

```diff
@@ -17612,10 +17612,14 @@ SubClassOf(Annotation(oboInOwl:is_inferred "true"^^xsd:string) obo:CL_0002323 ob
 AnnotationAssertion(Annotation(oboInOwl:hasDbXref "GOC:tfm"^^xsd:string) Annotation(oboInOwl:hasDbXref "PMID:11250738"^^xsd:string) Annotation(oboInOwl:hasDbXref "PMID:19022771"^^xsd:string) obo:IAO_0000115 obo:CL_0002324 "A myoepithelial cell that surrounds milk-secreting luminal epithelial cells. This cell type contracts under the stimulation of oxytocin and is attached to the basement membrane by hemidesmosomes and to the adjacent luminal cells by desmosomes. This cell type expresses high level of keratin-14 and is estrogen receptor alpha negative."^^xsd:string)
 AnnotationAssertion(oboInOwl:created_by obo:CL_0002324 "tmeehan"^^xsd:string)
 AnnotationAssertion(oboInOwl:creation_date obo:CL_0002324 "2010-09-20T11:42:14Z"^^xsd:string)
+AnnotationAssertion(oboInOwl:hasDbXref obo:CL_0002324 "CALOHA:TS-2378"^^xsd:string)
+AnnotationAssertion(oboInOwl:hasDbXref obo:CL_0002324 "FMA:67800"^^xsd:string)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "FMA:67800"^^xsd:string) oboInOwl:hasExactSynonym obo:CL_0002324 "basal cell of lactiferous duct"^^xsd:string)
+AnnotationAssertion(oboInOwl:hasExactSynonym obo:CL_0002324 "myoepithelial cell of lactiferous gland")
 AnnotationAssertion(oboInOwl:hasOBONamespace obo:CL_0002324 "cell"^^xsd:string)
 AnnotationAssertion(rdfs:label obo:CL_0002324 "myoepithelial cell of mammary gland"^^xsd:string)
-SubClassOf(obo:CL_0002324 obo:CL_0000185)
-SubClassOf(obo:CL_0002324 obo:CL_0002327)
+EquivalentClasses(obo:CL_0002324 ObjectIntersectionOf(obo:CL_0000185 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0001911)))
+SubClassOf(Annotation(oboInOwl:is_inferred "true"^^xsd:string) obo:CL_0002324 obo:CL_0000185)
```

Comment:
odd to assert this, but this may be legacy practice in CL

Reply:
yeah didn't notice that, don't know how I feel about that so I've removed it, thanks for the catch